base/applications/mspaint/lang/cs-CZ.rc

```diff
@@ -262,4 +262,5 @@ BEGIN
     IDS_VERTICAL "Vertical"
     IDS_PRINTRES "%d x %d pixel/cm"
     IDS_CANTPASTE "Failed to paste from the clipboard. The data format is either incorrect or not supported."
+    IDS_SAVEERROR "Failed to save an image as the following file:\r\n\r\n%s"
```

Comment:
What makes you think that \r\n is needed here? Is pure \n not sufficient?

Reply:
Fixed.